session/src/main/scala/akka/cassandra/session/javadsl/CassandraSessionRegistry.scala

```diff
@@ -55,7 +53,14 @@ final class CassandraSessionRegistry private (delegate: scaladsl.CassandraSessio
   def sessionFor(
       configPath: String,
       executionContext: ExecutionContext,
-      init: JFunction[CqlSession, CompletionStage[Done]]): CassandraSession = {
-    new CassandraSession(delegate.sessionFor(configPath, executionContext, ses => init(ses).toScala))
-  }
+      init: java.util.function.Function[CqlSession, CompletionStage[Done]]): CassandraSession =
```

Comment:
javadsl.CassandraSession isn't used consistently, but I don't understand why the javadsl should be used in the first place since that is the package of this class